README.md

```diff
@@ -25,6 +25,15 @@ First make sure you've installed all dependencies for building, as specified in
 the main Rust repository [here](https://github.com/rust-lang/rust/#building-from-source).
 Then use the following commands:
 
+### Notes for macOS
+
+The correct install directory for macOS is `/usr/local/avr-rust` rather than the Linux default `/opt/avr-rust`, so before the steps below, you must:
```

Comment:
"correct" is a debatable phrase — the correct place is wherever the user wants it to be installed. For example, I'd prefer to just install it into my home directory so as to avoid any permissions issues.

I'd prefer if this was relaxed to something suggesting "the Linux default won't work on macOS, if you want to install it, you need to change..."

FWIW, it's not needed to install it anyway; the compiler works just fine from the build directory. I do this to avoid taking up 2x the space.

Reply:
Due to [System Integrity Protection](https://support.apple.com/en-us/HT204899) normal users have less control over this on macOS than Linux, and `/usr/local` is explicitly designated as the place to install user-provided versions of things that are already installed on the macOS system (see "Scripting languages" [here](https://developer.apple.com/library/archive/documentation/Security/Conceptual/System_Integrity_Protection_Guide/FileSystemProtections/FileSystemProtections.html)), but I think it's reasonable to extend the analogy to a Rust cross-compiler.

Would you be happy with ~correct~ conventional?

Reply:
I too have struggled due to System Integrity Protection on Mac. I agree that recommending `/opt` for all cases disregards MacOS uses, this case deserves a mention.

I reckon we should avoid the vagueness and just directly parenthesize something like `(or, if you do not have write access to this directory, as is usually the case for MacOS, use something like `/usr/local/avr-rust`).